_layouts/post.html

```diff
@@ -408,6 +408,17 @@ <h3 class="author__name" itemprop="name">{{ author_details.name }}</h3>
     <div class="row">
       <div class="col-lg-8 col-md-12 mx-auto">
 
+        <!-- For the TOC -->
+        {% if page.toc %}
+          <aside class="sidebar__right">
+            <nav class="toc">
+              <header><h4 class="nav__title"><i class="fa fa-file-alt fa-w-12"></i> {{ page.toc_label | default: site.data.ui-text[site.locale].toc_label }}</h4></header>
+              {% include toc.html sanitize=true html=content h_min=2 h_max=3 class="toc__menu" %}
```

Comment:
Instead of hardcoding these values, you can take these as input from each post and if not provided, you can set 2 and 3 as min and max limits respectively.
You can take 
h_limits: [2, 3]
in the header and if not present then, set the custom values. Don't forget to add the min < max check.

Reply:
Hey @Signior-X , just this is left, I think you've done everything else :)